DataConnectors/FORCEPOINT_NGFW.json

```diff
@@ -0,0 +1,116 @@
+{
+  "id": "FORCEPOINT_NGFW",
+  "title": "Forcepoint NGFW",
+  "publisher": "Forcepoint",
+  "descriptionMarkdown": "The Forcepoint NGFW connector allows you to automatically export user-defined Forcepoint NGFW logs into Azure Sentinel in real-time. This enriches visibility into user activities recorded by NGFW, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel.​",
+  "graphQueries": [
+    {
+      "metricName": "Forcepoint NGFW log results",
+      "legend": "CommonSecurityLog",
+      "baseQuery": "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint\"\n| where DeviceProduct == \"NGFW\"\n"
+    }
+  ],
+  "sampleQueries": [
+    {
+      "description": "Show all terminated actions from the Forcepoint NGFW",
+      "query": "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint\"\n| where DeviceProduct == \"NGFW\"\n| where DeviceAction == \"Terminate\"\n"
+    },
+    {
+      "description": "Show all Forcepoint NGFW with suspected compromise behaviour",
+      "query": "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint\"\n| where DeviceProduct == \"NGFW\"\n| where Activity contains \"compromise\"\n"
+    },
+    {
+      "description": "Show chart grouping all Forcepoint NGFW events by Activity type",
+      "query": "\nCommonSecurityLog\n| where DeviceVendor == \"Forcepoint\"\n| where DeviceProduct == \"NGFW\"\n| summarize count=count() by Activity\n | render barchart\n"
+    }
+  ],
+  "dataTypes": [
+    {
+      "name": "CommonSecurityLog (CommonSecurityLog)",
```

Comment:
Update "name": "CommonSecurityLog (CommonSecurityLog)" to
"name": "CommonSecurityLog (ForcePointNGFW)"

Reply:
Updated